app/upgrades/v10/upgrades.go

```diff
@@ -0,0 +1,31 @@
+package v10
+
+import (
+	"context"
+
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+
+	"github.com/milkyway-labs/milkyway/v9/app/keepers"
+	ibchookstypes "github.com/milkyway-labs/milkyway/v9/x/ibc-hooks/types"
+)
+
+func CreateUpgradeHandler(
+	mm *module.Manager,
+	configuration module.Configurator,
+	keepers *keepers.AppKeepers,
+) upgradetypes.UpgradeHandler {
+	return func(ctx context.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		vm, err := mm.RunMigrations(ctx, configuration, fromVM)
+		if err != nil {
+			return nil, err
+		}
+
+		// Set the default ibchooks parameters
+		sdkCtx := sdk.UnwrapSDKContext(ctx)
+		keepers.IBCHooksKeeper.SetParams(sdkCtx, ibchookstypes.DefaultParams())
```

Comment:
This upgrade handler has been added to set the default modules parameters once we update the chain